components/network/latency/component.go

```diff
@@ -104,21 +110,34 @@ func (c *component) CheckOnce() {
 		c.lastMu.Unlock()
 	}()
 
-	cctx, ccancel := context.WithTimeout(c.ctx, 30*time.Second)
+	cctx, ccancel := context.WithTimeout(c.ctx, time.Minute)
```

Comment:
I'm curious why we change the timeout from 30s to 60s? 
```suggestion
	ctx, cancel := context.WithTimeout(c.ctx, time.Minute)
```